IFC4x3/Properties/s/SpeedLimit_3aq_ouBALD9vKxF2bvZwzE/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SpeedLimit_3aq_ouBALD9vKxF2bvZwzE" Name="SpeedLimit" UniqueId="e4d3ecb8-2ca5-4d27-953b-3c29798faf4e" PrimaryDataType="IfcLinearVelocityMeasure" />
```

Comment:
This one is confusing the description implies multiple values depending on other factors. I think a wider discussion is needed on how to handle Speed limit definitions. this relates to open issues on specifying design speed for road and rail sections.